src/types/Alert.ts

```diff
@@ -0,0 +1,13 @@
+import { AlertColor } from "@mui/material/Alert";
+import { AlertPropsColorOverrides } from "@mui/material/Alert/Alert";
+import { OverridableStringUnion } from "@mui/types";
+
+export type AlertSeverity = OverridableStringUnion<
+  AlertColor,
+  AlertPropsColorOverrides
+>;
+
+export default interface Alert {
+  message: string;
+  type: AlertSeverity;
+}
```

Comment:
You may want to just combine all these types files int a single types.d.ts file. Or keep these interface definitions with the components that they are used with. I don't think this project will evolve into something that has too many types for a single type definition file.

Reply:
Done. Although the .d.ts file is causing some issues in tests so I just used a .ts file. I must admit that I'm not too familiar with .d.ts